### PR TITLE
Inbound Call and OAuth Example

### PR DESCRIPTION
**Pull request by jpsantosbh**
* Fix the server-side token request to include the application-id 
* Adds the invite via push notification flow to the *Full* example use-case.
* Adds a OAuth flow request token exemple

### CODE REVIEW

File: public/full.js [hunk 1]
```diff
@@ -74,6 +75,136 @@ window.playbackEnded = () => {
...
+
+function b642ab(base64_string) {
+  return Uint8Array.from(window.atob(base64_string), c => c.charCodeAt(0));
+}
+
+async function readPushNotification(payload) {
+  console.log('payload', payload)
+
+  const key = b642ab(pnSecretKey)
+  // console.log('key', key)
```
Comment (iAmmar7) [inline public/full.js:176]:
```suggestion
```

File: public/full.js [hunk 2]
```diff
@@ -74,6 +75,136 @@ window.playbackEnded = () => {
...
+  const iv = b642ab(payload.iv)
+  // console.log('iv', iv)
```
Comment (iAmmar7) [inline public/full.js:178]:
```suggestion
```

File: public/full.js [hunk 3]
```diff
@@ -74,6 +75,136 @@ window.playbackEnded = () => {
...
+
+  // Chain invite and tag to have the full enc string
+  const full = atob(payload.invite) + atob(payload.tag)
+  const fullEncrypted = Uint8Array.from(full, (c) => c.charCodeAt(0))
+  // console.log('fullEncrypted', fullEncrypted)
```
Comment (iAmmar7) [inline public/full.js:183]:
```suggestion
```

File: public/full.js [hunk 4]
```diff
@@ -74,6 +75,136 @@ window.playbackEnded = () => {
...
+      keyData,
+      { name: 'AES-GCM' },
+      false,
+      ['decrypt']
+    )
+    return window.crypto.subtle.decrypt({ name: 'AES-GCM', iv }, key, data)
+  }
+
+  const compressed = await decrypt(key, iv, fullEncrypted)
+  // console.log('compressed', compressed)
```
Comment (iAmmar7) [inline public/full.js:197]:
```suggestion
```

File: views/minimal.ejs [hunk 5]
```diff
@@ -1,57 +1,65 @@
...
+      getToken,
+      onMessage,
+    }
+  </script>
+  <script type="text/javascript">
+    const _token = "<%= token %>";
+    const _host = "<%= host %>";
+    const _firebaseConfig = <% - firebaseConfig %>;
+  </script>
+  <script type="text/javascript" src="https://unpkg.com/@signalwire/js@dev"></script>
```
Comment (iAmmar7) [inline views/minimal.ejs:61]: Same as above

**Review by iAmmar7 — CHANGES_REQUESTED** [review]

File: views/index.ejs [hunk 6]
```diff
@@ -1,470 +1,308 @@
...
+      getToken,
+      onMessage,
+    }
+  </script>
+  <script type="text/javascript">
+    const _token = "<%= token %>";
+    const _host = "<%= host %>";
+    const _firebaseConfig = <%- firebaseConfig %>;
+  </script>
+  <script type="text/javascript" src="https://unpkg.com/@signalwire/js@dev"></script>
```
[the remark this answers is not in the document]
Comment (jpsantosbh) [reply views/index.ejs:304]: The demo is using the Call Fabric Browser SDK, not been released yet.

[the remark this answers is not in the document]
Comment (iAmmar7) [reply views/index.ejs:304]: Call Fabric has already been released, you can use it without the dev tag.

[the remark this answers is not in the document]
Comment (jpsantosbh) [reply views/index.ejs:304]: True, I was missing that thanks

**Review by iAmmar7 — APPROVED** [review]

**Pull request merged**